addons/binding/org.openhab.binding.network/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: Network Binding
 Bundle-SymbolicName: org.openhab.binding.network;singleton:=true
 Bundle-Vendor: openHAB
-Bundle-Version: 2.1.0.qualifier
+Bundle-Version: 2.1.1.qualifier
```

Comment:
I thought the same. Reminds me of another discussion thread where @kaikreuzer suggested to increment version numbers "more naturally".